### PR TITLE
fix(keepalive): migrated sensible early lifecycle and destroyed metho…

### LINKED ISSUE #922: Homepage backdrop isn't cleared when going to a library
**Steps To Reproduce**

1. Go to the home page and wait for the carousel to pop up
2. Navigate to a library
3. See that the home backdrop is still there

**Expected Behavior**

The background should go back to its default state

### PR DESCRIPTION
**Pull request by ThibaultNocchi**
…ds to activated/deactivated

fix #922

### CODE REVIEW

File: pages/fullscreen/playback/index.vue [hunk 1]
```diff
@@ -50,8 +50,7 @@ export default Vue.extend({
         keyboard: true,
         a11y: true
       } as SwiperOptions,
-      swiper: undefined as Swiper | undefined,
-      previousAppBarOpacity: 0
```
Comment (ferferga) [inline pages/fullscreen/playback/index.vue:54]: What if I'm accessing this page from an item details page, for instance? (As it has transparency)

Comment (ThibaultNocchi) [reply pages/fullscreen/playback/index.vue]: It seems every page that needs it set their transparency at activation, so it the player (or any page) shouldn't have to keep what was previously set imo :)

File: client/components/Buttons/LikeButton.vue [hunk 2]
```diff
@@ -23,7 +23,9 @@ export default Vue.extend({
   },
   data() {
     return {
-      isFavorite: false
+      isFavorite: false,
+      // eslint-disable-next-line @typescript-eslint/no-empty-function
+      unsubscribe(): void {}
```
Comment (ferferga) [inline client/components/Buttons/LikeButton.vue:28]: We won't need this as soon as the item store is properly implemented, that's why I intentionally skipped them in my PR. Doesn't harm to have this during the transition period.

**Review by ferferga — APPROVED** [review]

File: client/components/Buttons/LikeButton.vue (hunk 2 above)
Comment (ThibaultNocchi) [reply client/components/Buttons/LikeButton.vue:28]: Ok that's why :P

**Review by MrTimscampi — APPROVED** [review]

**Comment by MrTimscampi** [comment]
@jellyfin rebase

**Comment by MrTimscampi** [comment]
@jellyfin rebase

**Pull request merged**